File: authenticate/src/main/java/com/suntelecoms/authenticate/fingerprint/FingerPrintListener.java
package com.suntelecoms.authenticate.fingerprint;

/**
 * @Author Moustapha S. Dieme ( Djvmil_ ) on 3/12/20.
 */
public interface FingerPrintListener {

    void onSuccess();

    void onFailed();

    void onError(CharSequence errorString);

    void onHelp(CharSequence helpString);

}

File: authenticate/src/main/java/com/suntelecoms/authenticate/fingerprint/FingerPrintListener.kt
package com.suntelecoms.authenticate.fingerprint

/**
 * @Author Moustapha S. Dieme ( Djvmil_ ) on 3/12/20.
 */
interface FingerPrintListener {
    fun onSuccess()
    fun onFailed()
    fun onError(errorString: CharSequence?)
    fun onHelp(helpString: CharSequence?)
}